doc/tutorial.rst

```diff
@@ -1179,6 +1179,33 @@ Ocamlbuild
 
 There is an `atdgen plugin for ocamlbuild <https://github.com/hcarty/ocamlbuild-plugins/blob/master/myatdgen.ml>`__.
 
+Dune (jbuilder)
+---------------
+
+Dune currently needs atdgen build rules specified manually. Given an ``example.atd``,
+this will usually look like:
+
+.. code-block:: scheme
+
+  (rule
+   ((targets (example_j.ml
+              example_j.mli))
+    (deps    (example.atd))
+    (action  (run atdgen -j -j-std ${^}))))
+
+  (rule
+   ((targets (example_t.ml
+              example_t.mli))
+    (deps    (example.atd))
+    (action  (run atdgen -t -j-std ${^}))))
```

Comment:
You can omit the `-j-std` option here, it's only meaningful when used in conjunction with `-j`.